### PR TITLE
Removing src file

### PR DESCRIPTION
**Pull request by Rohit-Potnuru**
Remove src files

### CODE REVIEW

File: src/main/java/org/potrohit/springboot/PasteBinApplication/Main.java [hunk 1]
```diff
@@ -1,7 +0,0 @@
-package org.potrohit.springboot.PasteBinApplication;
-
-public class Main {
```
Comment (hsenid97) [inline src/main/java/org/potrohit/springboot/PasteBinApplication/Main.java:3]: why did u delete it

Comment (Rohit-Potnuru) [reply src/main/java/org/potrohit/springboot/PasteBinApplication/Main.java:3]: We do not need this file

Comment (hsenid97) [reply src/main/java/org/potrohit/springboot/PasteBinApplication/Main.java:3]: ok

**Review by abhinav135 — APPROVED** [review]

**Review by hsenid97 — APPROVED** [review]
ok approving for 1000$

**Pull request merged**